lib/src/installer/completion_installation.dart

```diff
@@ -265,6 +273,21 @@ class CompletionInstallation {
     );
   }
 
+  /// Tells the user to source the shell configuration file.
+  @visibleForTesting
+  void adviseSource(String rootCommand) {
+    final level = logger.level;
+    logger
+      ..level = Level.debug
```

Comment:
Why not just use info here?

Reply:
Changed to info because I was exploring the looks in detail and it sucks.